task_01.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Program to convert temperature in Fahrenheight to Celsius."""
+
+
+
+ABSOLUTE_DIFFERENCE = 273.15
+def farenheit_to_kelvin(degrees):
+
+
+    def celsius_to_kelvin(degrees):
```

Comment:
indentation here is off which makes this function unavailable to the module scope